include/items.php

```diff
@@ -167,6 +168,14 @@ function add_page_info_data($data) {
 
 	$text = "[attachment type='".$data["type"]."'";
 
+	if ($data["text"] == "") {
+		$data["text"] = $data["title"];
+	}
+
+	if ($data["text"] == "") {
+		$data["text"] = $data["url"];
+	}
+
 	if ($data["url"] != "")
```

Comment:
do you want to add the curly brackets here and further down along the lines as well? Just as the file is open already.